### PR TITLE
<bug Fix>Fix the opcua protocol creating too many clients, resulting in excessive session usage

### PR DESCRIPTION
**Pull request by leepala**
**What this PR does / why we need it**:
Fix the opcua protocol creating too many clients, resulting in excessive session usage
**Will this PR make the community happier**? 
yes!
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**How is this PR tested**
- [ ] unit test
- [ ] e2e test
- [x] other (please specify)
test by connect to s7 plc which turn on opcua server
**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/deviceshifu/deviceshifuopcua/deviceshifuopcua.go [hunk 1]
```diff
@@ -51,94 +51,41 @@ func New(deviceShifuMetadata *deviceshifubase.DeviceShifuMetaData) (*DeviceShifu
...
-					logger.Fatal("Cannot Get EndPoint Description")
-					return nil, err
-				}
 
-				ep := opcua.SelectEndpoint(endpoints, ua.SecurityPolicyURINone, ua.MessageSecurityModeNone)
-				if ep == nil {
-					logger.Fatal("Failed to find suitable endpoint")
-				}
+		ctx := context.Background()
+		opcuaClient, err = establishOPCUAConnection(ctx, *base.EdgeDevice.Spec.Address, base.EdgeDevice.Spec.ProtocolSettings.OPCUASetting)
```
Comment (tomqin93) [inline pkg/deviceshifu/deviceshifuopcua/deviceshifuopcua.go:62]: put this after `case v1alpha1.ProtocolOPCUA`?

**Review by tomqin93 — COMMENTED** [review]
@leepala added some comment, please take a look

File: pkg/deviceshifu/deviceshifuopcua/deviceshifuopcua.go [hunk 2]
```diff
@@ -155,6 +102,72 @@ func New(deviceShifuMetadata *deviceshifubase.DeviceShifuMetaData) (*DeviceShifu
...
+
+	switch ua.UserTokenTypeFromString(*setting.AuthenticationMode) {
+	case ua.UserTokenTypeIssuedToken:
+		options = append(options, opcua.AuthIssuedToken([]byte(*setting.IssuedToken)))
+	case ua.UserTokenTypeCertificate:
+		var privateKeyFileName = path.Join(DeviceConfigmapCertificatePath, *setting.PrivateKeyFileName)
+		var certificateFileName = path.Join(DeviceConfigmapCertificatePath, *setting.CertificateFileName)
+		cert, err := tls.LoadX509KeyPair(certificateFileName, privateKeyFileName)
+		if err != nil {
+			logger.Errorf("X509 Certificate Or PrivateKey load Default")
```
[the remark this answers is not in the document]
Comment (leepala) [reply pkg/deviceshifu/deviceshifuopcua/deviceshifuopcua.go:134]: cannot get certificate and the pod will crash
I will update the log

**Review by tomqin93 — APPROVED** [review]
LGTM

**Pull request merged**